R/summary-log.R

```diff
@@ -2,7 +2,8 @@
 #'
 #' Runs [model_summaries()] on all models in the input and returns a subset of the each resulting summary as a tibble.
 #'
-#' @return An object of class `bbi_summary_log_df`, which includes the fields described below.
+#' @return An object of class `bbi_summary_log_df`, which includes the fields described below. If _all_ model summaries
```

Comment:
The returned object will also include the `run` column, which is not described in `@details`.